package.json

```diff
@@ -6,7 +6,14 @@
   "scripts": {
     "prestart": "cp ./node_modules/react/umd/react.development.js dist/react.development.js && cp ./node_modules/react-dom/umd/react-dom.development.js dist/react-dom.development.js",
     "start": "webpack-dev-server --config ./webpack.config.js --mode development",
-    "test": "mocha -r ts-node/register src/**/*.test.ts"
+    "critter": "npm run tslint && npm run prettier && npm test",
+    "test": "mocha -r ts-node/register src/**/*.test.ts",
```

Comment:
Is this needed? I believe we got rid of Mocha and are now using Jest?? @katerberg